### PR TITLE
Update README.md2.1

### PR DESCRIPTION
**Pull request by jspooons**
ddd

### CODE REVIEW

**Review by jspoonerAC — COMMENTED** [review]
rr

File: README.md [hunk 1]
```diff
@@ -13,4 +13,6 @@ Cloning the repo and downloading the large datasets:
 1. Run `brew install lfs`
 2. Run `git lfs install` to initialize (or just run this command)
 3. Open new terminal
-4. Run `git clone <repository_url>`
\ No newline at end of file
+4. Run `git clone <repository_url>`
```
Comment (jspooons) [inline README.md:16]: test

Comment (jspooons) [inline README.md:16]: test 2

**Review by jspooons — COMMENTED** [review]
test 3

**Review by jspooons — COMMENTED** [review]
cc

**Pull request merged**